policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml

```diff
@@ -1,10 +1,12 @@
 description: Indiana provides this additional exemption amount for qualifying dependent children.
 values:
-  2021-01-01: 1_500
+  2021-01-01: 1_500 #first qualified year is 3_000
```

Comment:
remove comment, do we account for this in the code?

Reply:
We would add 
2023-01-01: 3_000

Add a birth year parameter and then check if the chidl is born after 2023 which then allows for a higher amount

Reply:
It looks like we didn't account for this in the code. I added a new parameter for this value but wasn't sure about how to modify the code, maybe we can talk about it on Monday